docs/11-circleci/03-build.mdx

```diff
@@ -0,0 +1,683 @@
+---
+toc_max_heading_level: 5
+---
+
+# Build
+
+The Unity orb provides the
+[build job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-build) to facilitate the
+process of building your Unity project to multiple platforms.
+
+### Parameters
+
+In this sub-section, you will find information on each parameter the job provides.
+
+#### build-name
+
+- _Default_: `UnityBuild`
+- _Required_: `False`
+- _Type_: `String`
+
+Your build's name. If you choose to compress your artifacts, this name will be used in the
+compressed file.
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          build-name: 'MyUnityProject'
+```
+
+#### build-target
+
+- _Default_: `N/A`
+- _Required_: `True`
+- _Type_: `String`
+
+The platform on which you want to run your game. Valid parameters can be found in the Unity Editor
+[documentation](https://docs.unity3d.com/2022.2/Documentation/ScriptReference/BuildTarget.html).
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          build-target: 'StandaloneLinux64'
+```
+
+#### compress
+
+- _Default_: `true`
+- _Required_: `False`
+- _Type_: `Boolean`
+
+Whether to compress the build output to a `.tar.gz` file. Set it to `true` if you want to download
+the build artifacts from the CircleCI dashboard, otherwise, you will have to download each file
+individually. If set to `false` for
+[decompressed WebGL](https://docs.unity3d.com/Manual/webgl-deploying.html) builds, the built project
+can be run directly from the GameCI dashboard.
```

Comment:
```suggestion
Whether to compress the build output to a `.tar.gz` file. Set it to `true` if you want to download
the build artifacts from the CircleCI dashboard. Otherwise, you will have to download each file
individually. If set to `false` for
[decompressed WebGL](https://docs.unity3d.com/Manual/webgl-deploying.html) builds, the built project
can be run directly from the GameCI dashboard.
```

Reply:
Done ✔️